test/mocha/10-test.js

```diff
@@ -45,3 +45,72 @@ describe('bedrock-jsonld-document-loader', () => {
     });
   });
 });
+
+describe('httpClientHandler', () => {
+  it('properly gets document', async () => {
+    let result;
+    let error;
+    const documentUrl = 'https://httpbin.org/json';
```

Comment:
Does it make more sense to have a local test server that can have various deterministic results to check?  Or is the idea to just use httpbin and if it ever stops working, then deal with it?

Reply:
I can update it to use custom local server if that's what you prefer, the reason why I used `httpbin` here was because I have seen it being used in other tests in `@digitalbazaar/http-client` lib and thought it was okay to use that here.

Reply:
Updated to use local server 6ca82c2 & 0a4ffe2